templates/ansible/roles/nodejs/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
@RDrop what are the benefits of installing latest version of nodejs manually? It's only needed as JS runtime for assets pipeline and I think version which is installed with apt-get works well.